packages/odyssey-react/src/components/Tooltip/Tooltip.theme.ts

```diff
@@ -13,20 +13,20 @@
 import type { ThemeReducer } from "@okta/odyssey-react-theme";
 
 export const theme: ThemeReducer = (theme) => ({
-  TipBorderRadius: theme.BorderRadiusBase,
-  TipPaddingBlock: theme.SpaceRemXs,
-  TipPaddingInline: theme.SpaceRemS,
-  TipTextColor: theme.ColorTextBodyInverse,
-  TipBackgroundColor: theme.ColorTextBody,
-  TipFontSize: theme.FontSizeCaption,
-  TipFontWeight: theme.FontWeightBold,
-  TipLineHeight: theme.FontLineHeightHeading,
-  TipBorderWidth: theme.BorderWidthBase,
```

Comment:
why remove the Tip token name part?

Reply:
The tokens are primarily used on the root and not a child element, so the prefix felt superfluous

Reply:
@andrewberg-okta thanks yea this is another tricky one, the whole what is a component part and what isn't discussion is another murky area for us. We'll just have to embrace some amount of ambiguity for now!